src/components/Button/Button.scss

```diff
@@ -0,0 +1,19 @@
+@use "../../assets/sass/variables" as *;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+.button {
+  font-family: $secondary-font-family;
+  background-color: $primary-color-green;
+  width: 300px;
```

Comment:
I think this should be responsive it will probably be worth setting it to 100%. As it will be a different sizes in different locations.

![image](https://user-images.githubusercontent.com/55165280/176466569-2b4434ae-25f8-400f-9655-deac43ec458e.png)
![image](https://user-images.githubusercontent.com/55165280/176466722-c1a967f1-1f4f-4992-8667-1fc046789d9d.png)

Reply:
I know it is technically against the conventions but in this case it is okay 👍